backend/src/main/java/com/bakdata/conquery/models/query/queryplan/aggregators/specific/value/ConceptValuesAggregator.java

```diff
@@ -0,0 +1,74 @@
+package com.bakdata.conquery.models.query.queryplan.aggregators.specific.value;
+
+import java.util.HashSet;
+import java.util.Optional;
+import java.util.Set;
+
+import com.bakdata.conquery.models.datasets.Column;
+import com.bakdata.conquery.models.datasets.Table;
+import com.bakdata.conquery.models.datasets.concepts.tree.ConceptTreeConnector;
+import com.bakdata.conquery.models.datasets.concepts.tree.TreeConcept;
+import com.bakdata.conquery.models.events.Bucket;
+import com.bakdata.conquery.models.query.QueryExecutionContext;
+import com.bakdata.conquery.models.query.entity.Entity;
+import com.bakdata.conquery.models.query.queryplan.aggregators.Aggregator;
+import com.bakdata.conquery.models.types.ResultType;
+import com.google.common.collect.ImmutableSet;
+import lombok.ToString;
+
+@ToString(callSuper = true, onlyExplicitlyIncluded = true)
+public class ConceptValuesAggregator extends Aggregator<Set<Object>> {
+
+	private final Set<Object> entries = new HashSet<>();
+	private final TreeConcept concept;
+
+	private Column column;
+
+	public ConceptValuesAggregator(TreeConcept concept) {
+		super();
+		this.concept = concept;
+	}
+
+	@Override
+	public void nextTable(QueryExecutionContext ctx, Table currentTable) {
+		Optional<ConceptTreeConnector> maybeConnector =
+				concept.getConnectors().stream()
+					   .filter(conn -> conn.getTable().equals(currentTable))
+					   .findAny();
+
+		if(maybeConnector.isEmpty()){
+			column = null;
+			return;
+		}
+
+
+		final ConceptTreeConnector connector = maybeConnector.get();
+
+		column = connector.getColumn();
+	}
```

Comment:
```suggestion
@ToString(callSuper = true, onlyExplicitlyIncluded = true)
public class ConceptValuesAggregator extends ColumnAggregator<Set<Object>> {

	private final Set<Object> entries = new HashSet<>();

        @Getter
        private final Column[] requiredColumns;
        
	public ConceptValuesAggregator(TreeConcept concept) {
		super();
		this.requiredColumns = concept.getConnectors().stream()
			   .map(ConceptTreeConnector::getColumn)
			   .filter(Object::notNull) ?
			   .toArray(Column[]::new);
	}
```

Reply:
requiredColumns macht eigentlich nichts außer ein bisschen Validierung, was erhoffst du dir von der Änderung?

Reply:
Mhmm ich dachte es macht noch etwas mehr, also vor auswahl mit is of interest, aber da habe ich mich glaube ich getäuscht.

Reply:
Ich hab gerade nochmal nachgeschaut, du hattest recht, die QPParentNode macht damit was über umwege